src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/CaseOfficerPreHearingHandler.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.presubmit;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.State;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PreSubmitCallbackStage;
+import uk.gov.hmcts.reform.iacaseapi.domain.handlers.PreSubmitCallbackHandler;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.*;
+
+@Component
+public class CaseOfficerPreHearingHandler implements PreSubmitCallbackHandler<AsylumCase> {
```

Comment:
There is a need for second Handler which implements PostSubmitCallHandler

Reply:
Yes - I removed the handler and the test which are not required.